Build native codes current master fails

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04 x86\_64): Ubuntu 22.04 on x64_64
- TensorFlow installed from (source or binary): source
- TensorFlow version: 2.15.0
- Java version (i.e., the output of `java -version`): openjdk 11
- Java command line flags (e.g., GC parameters): 
- Installed from Maven Central?: No
- Bazel version (if compiling from source): 6.1.0
- GCC/Compiler version (if compiling from source): clang 14.0.0-1ubuntu1.1
- CUDA/cuDNN version: -
- GPU model and memory: -

**Describe the problem**
Try to build the x86_64 native code jars for the current master of tensorflow/java (https://github.com/tensorflow/java/tree/2b6d83fd32ed1ec03e1e3468830a9ff12187fc1b)

**Provide the exact sequence of commands / steps that you executed before running into the problem**
Start ubuntu 22.04 docker image:
```
docker run -it -v $(pwd):/work -w /work ubuntu:22.04
```

Then prepare build environment for tensorflow in the running docker image:
```
TF_VERSION="v2.15.0"
apt update
apt install -y build-essential clang maven libgomp1
apt install -y python3-dev python3-pip
apt install -y pkg-config zip zlib1g-dev unzip curl tmux wget vim git htop
apt install -y openjdk-11-jdk
apt install -y liblapack3 libblas3 libhdf5-dev
pip3 install six mock numpy grpcio
pip3 install h5py
pip3 install keras_applications --no-deps
pip3 install keras_preprocessing --no-deps
pip3 install packaging

# get bazel version needed for the specific tensorflow version
git clone  https://github.com/tensorflow/tensorflow.git
cd tensorflow
git checkout "${TF_VERSION}"

# install needed bazel version
BAZEL_VERSION="$(cat .bazelversion|head -n1)"
wget https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-dist.zip
mkdir bazel-${BAZEL_VERSION}
unzip bazel-${BAZEL_VERSION}-dist.zip -d bazel-${BAZEL_VERSION}
cd bazel-${BAZEL_VERSION}
./compile.sh
cp output/bazel /usr/local/bin/
cd ..
```

Then try to build the native code jars like this:
```
TF_JAVA_VERSION="master"
git clone https://github.com/tensorflow/java.git
cd java
git checkout ${TF_JAVA_VERSION}
mvn clean install -Djavacpp.platform.host -Dnative.build
```

**Any other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

The run fails in the **TensorFlow Native Library** step in tensorflow java after successfully compiling tensorflow source:
```
[ . . . ]
[13,659 / 13,664] Compiling tensorflow/compiler/mlir/python/mlir.cc; 44s local ... (3 actions running)
[13,661 / 13,664] Compiling mlir/lib/Dialect/GPU/IR/GPUDialect.cpp; 18s local
[13,662 / 13,664] [Prepa] Linking external/org_tensorflow/tensorflow/libtensorflow_cc.so.2.15.0
[13,662 / 13,664] Linking external/org_tensorflow/tensorflow/libtensorflow_cc.so.2.15.0; 1s local
[13,663 / 13,664] [Prepa] action 'SolibSymlink _solib_k8/libexternal_Sorg_Utensorflow_Stensorflow_Slibtensorflow_Ucc.so.2.15.0'
Target @org_tensorflow//tensorflow:tensorflow_cc up-to-date:
  bazel-bin/external/org_tensorflow/tensorflow/libtensorflow_cc.so.2.15.0
INFO: Elapsed time: 2031.682s, Critical Path: 313.18s
INFO: 13664 processes: 900 internal, 12764 local.
INFO: Build completed successfully, 13664 total actions
Normalizing paths
Listing /java/tensorflow-core/tensorflow-core-native/bazel-bin/external/org_tensorflow/tensorflow:
total 608732
drwxr-xr-x  6 root root      4096 Mar 15 14:34 c
drwxr-xr-x  5 root root      4096 Mar 15 14:29 cc
drwxr-xr-x  7 root root      4096 Mar 15 14:25 compiler
drwxr-xr-x 28 root root      4096 Mar 15 14:48 core
drwxr-xr-x  3 root root      4096 Mar 15 14:22 distribute
drwxr-xr-x  5 root root      4096 Mar 15 14:21 dtensor
lrwxrwxrwx  1 root root        26 Mar 15 14:49 libtensorflow_cc.so -> libtensorflow_cc.so.2.15.0
lrwxrwxrwx  1 root root        26 Mar 15 14:49 libtensorflow_cc.so.2 -> libtensorflow_cc.so.2.15.0
-r-xr-xr-x  1 root root 574062848 Mar 15 14:49 libtensorflow_cc.so.2.15.0
-r-xr-xr-x  1 root root   1078929 Mar 15 14:15 libtensorflow_cc.so.2.15.0-2.params
lrwxrwxrwx  1 root root        33 Mar 15 14:49 libtensorflow_framework.so -> libtensorflow_framework.so.2.15.0
lrwxrwxrwx  1 root root        33 Mar 15 14:49 libtensorflow_framework.so.2 -> libtensorflow_framework.so.2.15.0
-r-xr-xr-x  1 root root  47893792 Mar 15 14:48 libtensorflow_framework.so.2.15.0
-r-xr-xr-x  1 root root    256786 Mar 15 14:15 libtensorflow_framework.so.2.15.0-2.params
drwxr-xr-x 15 root root      4096 Mar 15 14:16 lite
drwxr-xr-x  3 root root      4096 Mar 15 14:37 python
drwxr-xr-x  4 root root      4096 Mar 15 14:30 tools
[INFO] 
[INFO] --- exec-maven-plugin:3.1.0:exec (dist-download) @ tensorflow-core-native ---
[INFO] skipping execute as per configuration
[INFO] 
[INFO] --- maven-resources-plugin:3.3.1:resources (copy-resources) @ tensorflow-core-native ---
[INFO] skip non existing resourceDirectory /java/tensorflow-core/tensorflow-core-native/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.11.0:compile (javacpp-compiler) @ tensorflow-core-native ---
[INFO] Changes detected - recompiling the module! :source
[INFO] Compiling 1 source file with javac [forked debug release 8] to target/classes
[WARNING] Unable to autodetect 'javac' path, using 'javac' from the environment.
[INFO] 
[INFO] --- exec-maven-plugin:3.1.0:exec (bazel-generate) @ tensorflow-core-native ---
[INFO] skipping execute as per configuration
[INFO] 
[INFO] --- javacpp:1.5.10:parse (javacpp-parser) @ tensorflow-core-native ---
[INFO] Skipping execution of JavaCPP Builder
[INFO] 
[INFO] --- build-helper-maven-plugin:3.4.0:add-source (add-gen-sources) @ tensorflow-core-native ---
[INFO] Source directory: /java/tensorflow-core/tensorflow-core-native/src/gen/java added.
[INFO] 
[INFO] --- maven-resources-plugin:3.3.1:resources (default-resources) @ tensorflow-core-native ---
[INFO] skip non existing resourceDirectory /java/tensorflow-core/tensorflow-core-native/src/main/resources
[INFO] 
[INFO] --- maven-resources-plugin:3.3.1:copy-resources (generated-resources-copy) @ tensorflow-core-native ---
[INFO] skip non existing resourceDirectory /java/tensorflow-core/tensorflow-core-native/src/gen/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.11.0:compile (default-compile) @ tensorflow-core-native ---
[INFO] Changes detected - recompiling the module! :source
[INFO] Compiling 24 source files with javac [forked debug release 11 module-path] to target/classes
[WARNING] Unable to autodetect 'javac' path, using 'javac' from the environment.
[INFO] -------------------------------------------------------------
[WARNING] COMPILATION WARNING : 
[INFO] -------------------------------------------------------------
[WARNING] /java/tensorflow-core/tensorflow-core-native/src/main/java/module-info.java:[28,6] [module] module not found: tensorflow
[WARNING] /java/tensorflow-core/tensorflow-core-native/src/main/java/module-info.java:[18,34] requires transitive directive for an automatic module
[WARNING] /java/tensorflow-core/tensorflow-core-native/src/main/java/module-info.java:[19,32] requires transitive directive for an automatic module
[INFO] 3 warnings 
[INFO] -------------------------------------------------------------
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_SessionOptions.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_SessionOptions.java:[20] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_SessionOptions.java:[21,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_SessionOptions.java:[21] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_SessionOptions.java:[50,16] error: cannot find symbol
  symbol:   class TF_SessionOptions
  location: class AbstractTF_SessionOptions
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_SessionOptions.java:[28,51] error: cannot find symbol
  symbol:   class TF_SessionOptions
  location: class AbstractTF_SessionOptions
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_SessionOptions.java:[30,22] error: cannot find symbol
  symbol:   class TF_SessionOptions
  location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Graph.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Graph.java:[20] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Graph.java:[21,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Graph.java:[21] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Graph.java:[52,16] error: cannot find symbol
  symbol:   class TF_Graph
  location: class AbstractTF_Graph
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Graph.java:[28,51] error: cannot find symbol
  symbol:   class TF_Graph
  location: class AbstractTF_Graph
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Graph.java:[29,22] error: cannot find symbol
  symbol:   class TF_Graph
  location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_ContextOptions.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_ContextOptions.java:[20] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_ContextOptions.java:[21,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_ContextOptions.java:[21] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_ContextOptions.java:[50,16] error: cannot find symbol
  symbol:   class TFE_ContextOptions
  location: class AbstractTFE_ContextOptions
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_ContextOptions.java:[28,51] error: cannot find symbol
  symbol:   class TFE_ContextOptions
  location: class AbstractTFE_ContextOptions
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_ContextOptions.java:[30,22] error: cannot find symbol
  symbol:   class TFE_ContextOptions
  location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Function.java:[18,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Function.java:[18] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Function.java:[45,9] error: cannot find symbol
  symbol:   class TF_Function
  location: class AbstractTF_Function
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Function.java:[26,51] error: cannot find symbol
  symbol:   class TF_Function
  location: class AbstractTF_Function
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Function.java:[28,22] error: cannot find symbol
  symbol:   class TF_Function
  location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/TFJ_RuntimeLibrary.java:[22,43] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Buffer.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Buffer.java:[51,16] error: cannot find symbol
  symbol:   class TF_Buffer
  location: class AbstractTF_Buffer
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Buffer.java:[60,16] error: cannot find symbol
  symbol:   class TF_Buffer
  location: class AbstractTF_Buffer
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Buffer.java:[73,16] error: cannot find symbol
  symbol:   class TF_Buffer
  location: class AbstractTF_Buffer
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Buffer.java:[30,51] error: cannot find symbol
  symbol:   class TF_Buffer
  location: class AbstractTF_Buffer
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Buffer.java:[31,22] error: cannot find symbol
  symbol:   class TF_Buffer
  location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[40,12] error: cannot find symbol
  symbol:   class TF_Tensor
  location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[51,43] error: cannot find symbol
  symbol:   class TF_Tensor
  location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[51,56] error: cannot find symbol
  symbol:   class TF_Status
  location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[51,16] error: cannot find symbol
  symbol:   class TFE_TensorHandle
  location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[64,44] error: cannot find symbol
  symbol:   class TF_Status
  location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[64,9] error: cannot find symbol
  symbol:   class TFE_TensorHandle
  location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[77,39] error: cannot find symbol
  symbol:   class TFE_Context
  location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[77,76] error: cannot find symbol
  symbol:   class TF_Status
  location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[77,9] error: cannot find symbol
  symbol:   class TFE_TensorHandle
  location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[87,9] error: cannot find symbol
  symbol:   class TFE_TensorHandle
  location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[27,51] error: cannot find symbol
  symbol:   class TFE_TensorHandle
  location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[28,22] error: cannot find symbol
  symbol:   class TFE_TensorHandle
  location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Status.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Status.java:[56,16] error: cannot find symbol
  symbol:   class TF_Status
  location: class AbstractTF_Status
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Status.java:[35,51] error: cannot find symbol
  symbol:   class TF_Status
  location: class AbstractTF_Status
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Status.java:[36,22] error: cannot find symbol
  symbol:   class TF_Status
  location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[51,12] error: cannot find symbol
  symbol:   class TF_Graph
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[53,12] error: cannot find symbol
  symbol:   class TF_SessionOptions
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[54,12] error: cannot find symbol
  symbol:   class TF_Buffer
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[55,12] error: cannot find symbol
  symbol:   class TF_Buffer
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[56,12] error: cannot find symbol
  symbol:   class TF_Status
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[67,38] error: cannot find symbol
  symbol:   class TF_Graph
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[67,54] error: cannot find symbol
  symbol:   class TF_SessionOptions
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[67,78] error: cannot find symbol
  symbol:   class TF_Status
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[67,16] error: cannot find symbol
  symbol:   class TF_Session
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[84,6] error: cannot find symbol
  symbol:   class TF_SessionOptions
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[85,6] error: cannot find symbol
  symbol:   class TF_Buffer
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[88,6] error: cannot find symbol
  symbol:   class TF_Graph
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[89,6] error: cannot find symbol
  symbol:   class TF_Buffer
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[90,6] error: cannot find symbol
  symbol:   class TF_Status
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[83,16] error: cannot find symbol
  symbol:   class TF_Session
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[30,51] error: cannot find symbol
  symbol:   class TF_Session
  location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[31,22] error: cannot find symbol
  symbol:   class TF_Session
  location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Tensor.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Tensor.java:[58,19] error: cannot find symbol
  symbol:   class Deallocator_Pointer_long_Pointer
  location: class AbstractTF_Tensor
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Tensor.java:[76,16] error: cannot find symbol
  symbol:   class TF_Tensor
  location: class AbstractTF_Tensor
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Tensor.java:[91,16] error: cannot find symbol
  symbol:   class TF_Tensor
  location: class AbstractTF_Tensor
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Tensor.java:[108,9] error: cannot find symbol
  symbol:   class TF_Tensor
  location: class AbstractTF_Tensor
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Tensor.java:[27,51] error: cannot find symbol
  symbol:   class TF_Tensor
  location: class AbstractTF_Tensor
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Tensor.java:[28,22] error: cannot find symbol
  symbol:   class TF_Tensor
  location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[20] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[21,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[21] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[41,12] error: cannot find symbol
  symbol:   class TFE_ContextOptions
  location: class AbstractTFE_Context
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[52,39] error: cannot find symbol
  symbol:   class TFE_ContextOptions
  location: class AbstractTFE_Context
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[52,64] error: cannot find symbol
  symbol:   class TF_Status
  location: class AbstractTFE_Context
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[52,16] error: cannot find symbol
  symbol:   class TFE_Context
  location: class AbstractTFE_Context
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[28,51] error: cannot find symbol
  symbol:   class TFE_Context
  location: class AbstractTFE_Context
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[29,22] error: cannot find symbol
  symbol:   class TFE_Context
  location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_ImportGraphDefOptions.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_ImportGraphDefOptions.java:[20] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_ImportGraphDefOptions.java:[21,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_ImportGraphDefOptions.java:[21] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_ImportGraphDefOptions.java:[50,16] error: cannot find symbol
  symbol:   class TF_ImportGraphDefOptions
  location: class AbstractTF_ImportGraphDefOptions
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_ImportGraphDefOptions.java:[28,51] error: cannot find symbol
  symbol:   class TF_ImportGraphDefOptions
  location: class AbstractTF_ImportGraphDefOptions
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_ImportGraphDefOptions.java:[30,22] error: cannot find symbol
  symbol:   class TF_ImportGraphDefOptions
  location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[20] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[21,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[21] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[41,12] error: cannot find symbol
  symbol:   class TFE_Context
  location: class AbstractTFE_Op
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[52,29] error: cannot find symbol
  symbol:   class TFE_Context
  location: class AbstractTFE_Op
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[52,74] error: cannot find symbol
  symbol:   class TF_Status
  location: class AbstractTFE_Op
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[52,16] error: cannot find symbol
  symbol:   class TFE_Op
  location: class AbstractTFE_Op
[INFO] 100 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for TensorFlow Java Parent 1.0.0-SNAPSHOT:
[INFO] 
[INFO] TensorFlow Java Parent ............................. SUCCESS [  0.568 s]
[INFO] TensorFlow Core Parent ............................. SUCCESS [  0.009 s]
[INFO] TensorFlow Native Library .......................... FAILURE [33:55 min]
[INFO] TensorFlow Generators .............................. SKIPPED
[INFO] TensorFlow API ..................................... SKIPPED
[INFO] TensorFlow API Platform ............................ SKIPPED
[INFO] TensorFlow Framework Library ....................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  33:56 min
[INFO] Finished at: 2024-03-15T14:49:08Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project tensorflow-core-native: Compilation failure: Compilation failure: 
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_SessionOptions.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_SessionOptions.java:[20] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_SessionOptions.java:[21,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_SessionOptions.java:[21] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_SessionOptions.java:[50,16] error: cannot find symbol
[ERROR]   symbol:   class TF_SessionOptions
[ERROR]   location: class AbstractTF_SessionOptions
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_SessionOptions.java:[28,51] error: cannot find symbol
[ERROR]   symbol:   class TF_SessionOptions
[ERROR]   location: class AbstractTF_SessionOptions
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_SessionOptions.java:[30,22] error: cannot find symbol
[ERROR]   symbol:   class TF_SessionOptions
[ERROR]   location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Graph.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Graph.java:[20] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Graph.java:[21,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Graph.java:[21] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Graph.java:[52,16] error: cannot find symbol
[ERROR]   symbol:   class TF_Graph
[ERROR]   location: class AbstractTF_Graph
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Graph.java:[28,51] error: cannot find symbol
[ERROR]   symbol:   class TF_Graph
[ERROR]   location: class AbstractTF_Graph
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Graph.java:[29,22] error: cannot find symbol
[ERROR]   symbol:   class TF_Graph
[ERROR]   location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_ContextOptions.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_ContextOptions.java:[20] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_ContextOptions.java:[21,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_ContextOptions.java:[21] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_ContextOptions.java:[50,16] error: cannot find symbol
[ERROR]   symbol:   class TFE_ContextOptions
[ERROR]   location: class AbstractTFE_ContextOptions
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_ContextOptions.java:[28,51] error: cannot find symbol
[ERROR]   symbol:   class TFE_ContextOptions
[ERROR]   location: class AbstractTFE_ContextOptions
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_ContextOptions.java:[30,22] error: cannot find symbol
[ERROR]   symbol:   class TFE_ContextOptions
[ERROR]   location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Function.java:[18,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Function.java:[18] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Function.java:[45,9] error: cannot find symbol
[ERROR]   symbol:   class TF_Function
[ERROR]   location: class AbstractTF_Function
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Function.java:[26,51] error: cannot find symbol
[ERROR]   symbol:   class TF_Function
[ERROR]   location: class AbstractTF_Function
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Function.java:[28,22] error: cannot find symbol
[ERROR]   symbol:   class TF_Function
[ERROR]   location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/TFJ_RuntimeLibrary.java:[22,43] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Buffer.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Buffer.java:[51,16] error: cannot find symbol
[ERROR]   symbol:   class TF_Buffer
[ERROR]   location: class AbstractTF_Buffer
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Buffer.java:[60,16] error: cannot find symbol
[ERROR]   symbol:   class TF_Buffer
[ERROR]   location: class AbstractTF_Buffer
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Buffer.java:[73,16] error: cannot find symbol
[ERROR]   symbol:   class TF_Buffer
[ERROR]   location: class AbstractTF_Buffer
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Buffer.java:[30,51] error: cannot find symbol
[ERROR]   symbol:   class TF_Buffer
[ERROR]   location: class AbstractTF_Buffer
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Buffer.java:[31,22] error: cannot find symbol
[ERROR]   symbol:   class TF_Buffer
[ERROR]   location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[40,12] error: cannot find symbol
[ERROR]   symbol:   class TF_Tensor
[ERROR]   location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[51,43] error: cannot find symbol
[ERROR]   symbol:   class TF_Tensor
[ERROR]   location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[51,56] error: cannot find symbol
[ERROR]   symbol:   class TF_Status
[ERROR]   location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[51,16] error: cannot find symbol
[ERROR]   symbol:   class TFE_TensorHandle
[ERROR]   location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[64,44] error: cannot find symbol
[ERROR]   symbol:   class TF_Status
[ERROR]   location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[64,9] error: cannot find symbol
[ERROR]   symbol:   class TFE_TensorHandle
[ERROR]   location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[77,39] error: cannot find symbol
[ERROR]   symbol:   class TFE_Context
[ERROR]   location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[77,76] error: cannot find symbol
[ERROR]   symbol:   class TF_Status
[ERROR]   location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[77,9] error: cannot find symbol
[ERROR]   symbol:   class TFE_TensorHandle
[ERROR]   location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[87,9] error: cannot find symbol
[ERROR]   symbol:   class TFE_TensorHandle
[ERROR]   location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[27,51] error: cannot find symbol
[ERROR]   symbol:   class TFE_TensorHandle
[ERROR]   location: class AbstractTFE_TensorHandle
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_TensorHandle.java:[28,22] error: cannot find symbol
[ERROR]   symbol:   class TFE_TensorHandle
[ERROR]   location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Status.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Status.java:[56,16] error: cannot find symbol
[ERROR]   symbol:   class TF_Status
[ERROR]   location: class AbstractTF_Status
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Status.java:[35,51] error: cannot find symbol
[ERROR]   symbol:   class TF_Status
[ERROR]   location: class AbstractTF_Status
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Status.java:[36,22] error: cannot find symbol
[ERROR]   symbol:   class TF_Status
[ERROR]   location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[51,12] error: cannot find symbol
[ERROR]   symbol:   class TF_Graph
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[53,12] error: cannot find symbol
[ERROR]   symbol:   class TF_SessionOptions
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[54,12] error: cannot find symbol
[ERROR]   symbol:   class TF_Buffer
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[55,12] error: cannot find symbol
[ERROR]   symbol:   class TF_Buffer
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[56,12] error: cannot find symbol
[ERROR]   symbol:   class TF_Status
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[67,38] error: cannot find symbol
[ERROR]   symbol:   class TF_Graph
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[67,54] error: cannot find symbol
[ERROR]   symbol:   class TF_SessionOptions
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[67,78] error: cannot find symbol
[ERROR]   symbol:   class TF_Status
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[67,16] error: cannot find symbol
[ERROR]   symbol:   class TF_Session
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[84,6] error: cannot find symbol
[ERROR]   symbol:   class TF_SessionOptions
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[85,6] error: cannot find symbol
[ERROR]   symbol:   class TF_Buffer
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[88,6] error: cannot find symbol
[ERROR]   symbol:   class TF_Graph
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[89,6] error: cannot find symbol
[ERROR]   symbol:   class TF_Buffer
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[90,6] error: cannot find symbol
[ERROR]   symbol:   class TF_Status
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[83,16] error: cannot find symbol
[ERROR]   symbol:   class TF_Session
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[30,51] error: cannot find symbol
[ERROR]   symbol:   class TF_Session
[ERROR]   location: class AbstractTF_Session
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Session.java:[31,22] error: cannot find symbol
[ERROR]   symbol:   class TF_Session
[ERROR]   location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Tensor.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Tensor.java:[58,19] error: cannot find symbol
[ERROR]   symbol:   class Deallocator_Pointer_long_Pointer
[ERROR]   location: class AbstractTF_Tensor
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Tensor.java:[76,16] error: cannot find symbol
[ERROR]   symbol:   class TF_Tensor
[ERROR]   location: class AbstractTF_Tensor
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Tensor.java:[91,16] error: cannot find symbol
[ERROR]   symbol:   class TF_Tensor
[ERROR]   location: class AbstractTF_Tensor
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Tensor.java:[108,9] error: cannot find symbol
[ERROR]   symbol:   class TF_Tensor
[ERROR]   location: class AbstractTF_Tensor
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Tensor.java:[27,51] error: cannot find symbol
[ERROR]   symbol:   class TF_Tensor
[ERROR]   location: class AbstractTF_Tensor
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_Tensor.java:[28,22] error: cannot find symbol
[ERROR]   symbol:   class TF_Tensor
[ERROR]   location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[20] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[21,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[21] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[41,12] error: cannot find symbol
[ERROR]   symbol:   class TFE_ContextOptions
[ERROR]   location: class AbstractTFE_Context
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[52,39] error: cannot find symbol
[ERROR]   symbol:   class TFE_ContextOptions
[ERROR]   location: class AbstractTFE_Context
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[52,64] error: cannot find symbol
[ERROR]   symbol:   class TF_Status
[ERROR]   location: class AbstractTFE_Context
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[52,16] error: cannot find symbol
[ERROR]   symbol:   class TFE_Context
[ERROR]   location: class AbstractTFE_Context
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[28,51] error: cannot find symbol
[ERROR]   symbol:   class TFE_Context
[ERROR]   location: class AbstractTFE_Context
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Context.java:[29,22] error: cannot find symbol
[ERROR]   symbol:   class TFE_Context
[ERROR]   location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_ImportGraphDefOptions.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_ImportGraphDefOptions.java:[20] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_ImportGraphDefOptions.java:[21,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_ImportGraphDefOptions.java:[21] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_ImportGraphDefOptions.java:[50,16] error: cannot find symbol
[ERROR]   symbol:   class TF_ImportGraphDefOptions
[ERROR]   location: class AbstractTF_ImportGraphDefOptions
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_ImportGraphDefOptions.java:[28,51] error: cannot find symbol
[ERROR]   symbol:   class TF_ImportGraphDefOptions
[ERROR]   location: class AbstractTF_ImportGraphDefOptions
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTF_ImportGraphDefOptions.java:[30,22] error: cannot find symbol
[ERROR]   symbol:   class TF_ImportGraphDefOptions
[ERROR]   location: class DeleteDeallocator
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[20,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[20] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[21,50] error: package org.tensorflow.internal.c_api.global does not exist
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[21] error: static import only from classes and interfaces
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[41,12] error: cannot find symbol
[ERROR]   symbol:   class TFE_Context
[ERROR]   location: class AbstractTFE_Op
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[52,29] error: cannot find symbol
[ERROR]   symbol:   class TFE_Context
[ERROR]   location: class AbstractTFE_Op
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[52,74] error: cannot find symbol
[ERROR]   symbol:   class TF_Status
[ERROR]   location: class AbstractTFE_Op
[ERROR] /java/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/AbstractTFE_Op.java:[52,16] error: cannot find symbol
[ERROR]   symbol:   class TFE_Op
[ERROR]   location: class AbstractTFE_Op
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :tensorflow-core-native
```

Did I do something wrong? Or is this simply the current state of the repo, and needs fixes for the javaCPP bindings?

Thanks for your reply!

When I run `mvn clean install -e -Djavacpp.platform.host -Dnative.build -Djavacpp.generate.skip=false` it fails after the native build with this error:
```
[ ... ]
[13,651 / 13,664] Compiling tensorflow/compiler/mlir/python/mlir.cc; 39s local ... (11 actions running)
[13,655 / 13,664] Compiling tensorflow/compiler/mlir/python/mlir.cc; 41s local ... (7 actions running)
[13,657 / 13,664] Compiling tensorflow/compiler/mlir/python/mlir.cc; 43s local ... (5 actions running)
[13,661 / 13,664] Compiling tensorflow/compiler/mlir/python/mlir.cc; 44s local
[13,662 / 13,664] Linking external/org_tensorflow/tensorflow/libtensorflow_cc.so.2.15.0; 1s local
[13,663 / 13,664] checking cached actions
Target @org_tensorflow//tensorflow:tensorflow_cc up-to-date:
  bazel-bin/external/org_tensorflow/tensorflow/libtensorflow_cc.so.2.15.0
INFO: Elapsed time: 2060.587s, Critical Path: 295.20s
INFO: 13664 processes: 900 internal, 12764 local.
INFO: Build completed successfully, 13664 total actions
Normalizing paths
Listing /java/tensorflow-core/tensorflow-core-native/bazel-bin/external/org_tensorflow/tensorflow:
total 608732
drwxr-xr-x  6 root root      4096 Mar 18 09:01 c
drwxr-xr-x  5 root root      4096 Mar 18 09:04 cc
drwxr-xr-x  7 root root      4096 Mar 18 09:22 compiler
drwxr-xr-x 28 root root      4096 Mar 18 09:28 core
drwxr-xr-x  3 root root      4096 Mar 18 09:08 distribute
drwxr-xr-x  5 root root      4096 Mar 18 09:00 dtensor
lrwxrwxrwx  1 root root        26 Mar 18 09:29 libtensorflow_cc.so -> libtensorflow_cc.so.2.15.0
lrwxrwxrwx  1 root root        26 Mar 18 09:29 libtensorflow_cc.so.2 -> libtensorflow_cc.so.2.15.0
-r-xr-xr-x  1 root root 574062848 Mar 18 09:29 libtensorflow_cc.so.2.15.0
-r-xr-xr-x  1 root root   1078929 Mar 18 08:56 libtensorflow_cc.so.2.15.0-2.params
lrwxrwxrwx  1 root root        33 Mar 18 09:29 libtensorflow_framework.so -> libtensorflow_framework.so.2.15.0
lrwxrwxrwx  1 root root        33 Mar 18 09:29 libtensorflow_framework.so.2 -> libtensorflow_framework.so.2.15.0
-r-xr-xr-x  1 root root  47893792 Mar 18 09:29 libtensorflow_framework.so.2.15.0
-r-xr-xr-x  1 root root    256786 Mar 18 08:56 libtensorflow_framework.so.2.15.0-2.params
drwxr-xr-x 15 root root      4096 Mar 18 09:00 lite
drwxr-xr-x  3 root root      4096 Mar 18 09:13 python
drwxr-xr-x  4 root root      4096 Mar 18 09:08 tools
[INFO] 
[INFO] --- exec-maven-plugin:3.1.0:exec (dist-download) @ tensorflow-core-native ---
[INFO] skipping execute as per configuration
[INFO] 
[INFO] --- maven-resources-plugin:3.3.1:resources (copy-resources) @ tensorflow-core-native ---
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/shared/maven-filtering/3.3.1/maven-filtering-3.3.1.pom
 [ ... ]
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/errorprone/javac/9+181-r4173-1/javac-9+181-r4173-1.jar (6.9 MB at 8.4 MB/s)
[INFO] Changes detected - recompiling the module! :dependency
[INFO] Compiling 1 source file with javac [forked debug release 8] to target/classes
[WARNING] Unable to autodetect 'javac' path, using 'javac' from the environment.
[INFO] 
[INFO] --- exec-maven-plugin:3.1.0:exec (bazel-generate) @ tensorflow-core-native ---
[INFO] skipping execute as per configuration
[INFO] 
[INFO] --- javacpp:1.5.10:parse (javacpp-parser) @ tensorflow-core-native ---
[INFO] Detected platform "linux-x86_64"
[INFO] Building platform "linux-x86_64"
[INFO] Targeting /java/tensorflow-core/tensorflow-core-native/src/gen/java/org/tensorflow/internal/c_api/global/tensorflow.java
[ERROR] Failed to execute JavaCPP Builder: Could not parse "tensorflow/tsl/platform/ctstring_internal.h": File does not exist
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for TensorFlow Java Parent 1.0.0-SNAPSHOT:
[INFO] 
[INFO] TensorFlow Java Parent ............................. SUCCESS [ 11.030 s]
[INFO] TensorFlow Core Parent ............................. SUCCESS [  0.006 s]
[INFO] TensorFlow Native Library .......................... FAILURE [34:26 min]
[INFO] TensorFlow Generators .............................. SKIPPED
[INFO] TensorFlow API ..................................... SKIPPED
[INFO] TensorFlow API Platform ............................ SKIPPED
[INFO] TensorFlow Framework Library ....................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  34:37 min
[INFO] Finished at: 2024-03-18T09:29:45Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.bytedeco:javacpp:1.5.10:parse (javacpp-parser) on project tensorflow-core-native: Failed to execute JavaCPP Builder: Could not parse "tensorflow/tsl/platform/ctstring_internal.h": File does not exist -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.bytedeco:javacpp:1.5.10:parse (javacpp-parser) on project tensorflow-core-native: Failed to execute JavaCPP Builder
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: org.apache.maven.plugin.MojoExecutionException: Failed to execute JavaCPP Builder
    at org.bytedeco.javacpp.tools.BuildMojo.execute (BuildMojo.java:423)
    at org.bytedeco.javacpp.tools.ParseMojo.execute (ParseMojo.java:39)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: java.io.FileNotFoundException: Could not parse "tensorflow/tsl/platform/ctstring_internal.h": File does not exist
    at org.bytedeco.javacpp.tools.Parser.parse (Parser.java:4571)
    at org.bytedeco.javacpp.tools.Parser.parse (Parser.java:4749)
    at org.bytedeco.javacpp.tools.Builder.parse (Builder.java:95)
    at org.bytedeco.javacpp.tools.Builder.build (Builder.java:1095)
    at org.bytedeco.javacpp.tools.BuildMojo.execute (BuildMojo.java:417)
    at org.bytedeco.javacpp.tools.ParseMojo.execute (ParseMojo.java:39)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[ERROR] 
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :tensorflow-core-native
```

--- a/tensorflow-core/tensorflow-core-generator/src/main/java/org/tensorflow/generator/op/OpGenerator.java
+++ b/tensorflow-core/tensorflow-core-generator/src/main/java/org/tensorflow/generator/op/OpGenerator.java
@@ -37,6 +37,7 @@ import java.util.List;
 import java.util.Map;
 import java.util.Scanner;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
 import org.bytedeco.javacpp.BytePointer;
 import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
 import org.tensorflow.internal.c_api.TF_ApiDefMap;
@@ -72,7 +73,7 @@ public final class OpGenerator {
 
   private static final String DEFAULT_OP_DEF_FILE = "org/tensorflow/ops.pbtxt";
 
-  private static final Scanner USER_PROMPT = new Scanner(System.in);
+  private static final Scanner USER_PROMPT = new Scanner(System.in, StandardCharsets.UTF_8);
 
   /**
    * Args should be {@code <outputDir> <opDefFile> [base_package]}.
@@ -195,7 +196,8 @@ public final class OpGenerator {
   private static OpList readOpList(String filename, InputStream protoInput) {
     try {
       if (filename.endsWith(".pbtxt")) {
-        return TextFormat.parse(new String(protoInput.readAllBytes()), OpList.class);
+        return TextFormat.parse(
+            new String(protoInput.readAllBytes(), StandardCharsets.UTF_8), OpList.class);
       }
       return OpList.parseFrom(protoInput);
 
@@ -286,9 +288,8 @@ public final class OpGenerator {
   }
 
   private void mergeApiDefs(TF_ApiDefMap apiDefMap, TF_Status status) {
-    try {
-      Files.walk(apiDefsPath)
-          .filter(p -> p.toString().endsWith(".pbtxt"))
+    try (Stream<Path> s = Files.walk(apiDefsPath)) {
+      s.filter(p -> p.toString().endsWith(".pbtxt"))
           .forEach(
               p -> {
                 try {
@@ -366,7 +367,7 @@ public final class OpGenerator {
     if (!apiDefFile.exists() && !apiDefFile.createNewFile()) {
       System.err.println("Cannot create API definition file \"" + apiDefFile.getPath() + "\"");
     }
-    try (var apiDefWriter = new FileWriter(apiDefFile)) {
+    try (var apiDefWriter = new FileWriter(apiDefFile, StandardCharsets.UTF_8)) {
       var apiDefs = ApiDefs.newBuilder();
       apiDefs.addOp(apiDef.build());
       apiDefWriter.write(TextFormat.printer().printToString(apiDefs.build()));

--- a/tensorflow-core/tensorflow-core-generator/src/main/java/org/tensorflow/generator/op/javadoc/JavaDocNodeRendererContext.java
+++ b/tensorflow-core/tensorflow-core-generator/src/main/java/org/tensorflow/generator/op/javadoc/JavaDocNodeRendererContext.java
@@ -9,6 +9,8 @@ import org.commonmark.renderer.html.UrlSanitizer;
 public interface JavaDocNodeRendererContext {
 
   /**
+   * Encode a URL into a String.
+   *
    * @param url to be encoded
    * @return an encoded URL (depending on the configuration)
    */
@@ -26,11 +28,15 @@ public interface JavaDocNodeRendererContext {
   Map<String, String> extendAttributes(Node node, String tagName, Map<String, String> attributes);
 
   /**
+   * Gets the HTML writer.
+   *
    * @return the HTML writer to use
    */
   JavaDocWriter getWriter();
 
   /**
+   * The HTML for a line break.
+   *
    * @return HTML that should be rendered for a soft line break
    */
   String getSoftbreak();
@@ -45,17 +51,23 @@ public interface JavaDocNodeRendererContext {
   void render(Node node);
 
   /**
+   * Should HTML be escaped?
+   *
    * @return whether HTML blocks and tags should be escaped or not
    */
   boolean shouldEscapeHtml();
 
   /**
+   * Should URLs be sanitized?
+   *
    * @return true if the {@link UrlSanitizer} should be used.
    * @since 0.14.0
    */
   boolean shouldSanitizeUrls();
 
   /**
+   * Gets the URL sanitizer.
+   *
    * @return Sanitizer to use for securing {@link Link} href and {@link Image} src if {@link
    *     #shouldSanitizeUrls()} is true.
    * @since 0.14.0

--- a/tensorflow-core/tensorflow-core-native/pom.xml
+++ b/tensorflow-core/tensorflow-core-native/pom.xml
@@ -318,6 +318,7 @@
             <includePath>${native.source.directory}/org/tensorflow/internal/c_api/</includePath>
             <!-- additional include paths in case of a full native build -->
             <includePath>${project.basedir}/bazel-${project.artifactId}/external/org_tensorflow/</includePath>
+            <includePath>${project.basedir}/bazel-${project.artifactId}/external/local_tsl/</includePath>
             <includePath>${project.basedir}/bazel-bin/external/org_tensorflow/</includePath>
             <includePath>${project.basedir}/bazel-${project.artifactId}/external/com_google_absl/</includePath>
             <includePath>${project.basedir}/bazel-${project.artifactId}/external/eigen_archive/</includePath>

--- a/tensorflow-core/tensorflow-core-native/src/gen/java/org/tensorflow/internal/c_api/global/tensorflow.java
+++ b/tensorflow-core/tensorflow-core-native/src/gen/java/org/tensorflow/internal/c_api/global/tensorflow.java
@@ -11,7 +11,7 @@ import org.bytedeco.javacpp.annotation.*;
 public class tensorflow extends org.tensorflow.internal.c_api.presets.tensorflow {
     static { Loader.load(); }
 
-// Parsed from tensorflow/tsl/platform/ctstring_internal.h
+// Parsed from tsl/platform/ctstring_internal.h
 
 /* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
 
@@ -165,7 +165,7 @@ public static native void TF_TString_Move(TF_TString dst, TF_TString src);
 // #endif  // TENSORFLOW_TSL_PLATFORM_CTSTRING_INTERNAL_H_
 
 
-// Parsed from tensorflow/tsl/platform/ctstring.h
+// Parsed from tsl/platform/ctstring.h
 
 /* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
 
@@ -274,7 +274,7 @@ limitations under the License.
 // #endif  // TENSORFLOW_TSL_PLATFORM_CTSTRING_H_
 
 
-// Parsed from tensorflow/tsl/c/tsl_status.h
+// Parsed from tsl/c/tsl_status.h
 
 /* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
 

--- a/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/presets/tensorflow.java
+++ b/tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/presets/tensorflow.java
@@ -36,9 +36,11 @@ import org.bytedeco.javacpp.tools.InfoMapper;
           value = {"linux", "macosx", "windows"},
           compiler = "cpp17",
           include = {
-            "tensorflow/tsl/platform/ctstring_internal.h",
-            "tensorflow/tsl/platform/ctstring.h",
-            "tensorflow/tsl/c/tsl_status.h",
+            // TSL headers are in different places in a bazel build and the downloaded whl
+            // The lower part is still the same, so multiple roots are set in the pom file.
+            "tsl/platform/ctstring_internal.h",
+            "tsl/platform/ctstring.h",
+            "tsl/c/tsl_status.h",
             "tensorflow/c/c_api_macros.h",
             "tensorflow/c/tf_datatype.h",
             "tensorflow/c/tf_status.h",

